added updated index.html

shanecoughlan opened the pull request:
<img width="1512" alt="Screen Shot 2019-04-26 at 16 04 04" src="https://user-images.githubusercontent.com/3616071/56789727-8da1dd80-683d-11e9-91e2-fb5e066e568b.png">
Suggested major update including a video gate.

shanecoughlan commented:
@goneall, thanks!

@brandelune, do we still have translation available? If so, can you update https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/tree/master/WebContent/resources/locales/ja as a pull request to reflect the new text in this thread?

Here is the direct link to the English file in the merge request so you do not need to scroll in this thread:
https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/pull/146/commits/7655bc237c6e7816641cdda0f961bb5c460b8683

brandelune commented:
Definitely !

I'll check that later today !

JC

> On Apr 30, 2019, at 7:54, Shane Coughlan <notifications@github.com <mailto:notifications@github.com>> wrote:
> 
> @goneall <https://github.com/goneall>, thanks!
> 
> @brandelune <https://github.com/brandelune>, do we still have translation available? If so, can you update https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/tree/master/WebContent/resources/locales/ja <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/tree/master/WebContent/resources/locales/ja> as a pull request to reflect the new text in this thread?
> 
> Here is the direct link to the English file in the merge request so you do not need to scroll in this thread:
> 7655bc2 <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/commit/7655bc237c6e7816641cdda0f961bb5c460b8683>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/pull/146#issuecomment-487771732>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AG427GE5D3V3IMLCHKHXFNDPS54CDANCNFSM4HITBX7A>.
> 

Jean-Christophe Helary
-----------------------------------------------
http://mac4translators.blogspot.com <http://mac4translators.blogspot.com/> @brandelune

shanecoughlan commented:
> Definitely ! I'll check that later today ! JC
> On Apr 30, 2019, at 7:54, Shane Coughlan ***@***.*** ***@***.***>> wrote: @goneall <https://github.com/goneall>, thanks! @brandelune <https://github.com/brandelune>, do we still have translation available? If so, can you update https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/tree/master/WebContent/resources/locales/ja <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/tree/master/WebContent/resources/locales/ja> as a pull request to reflect the new text in this thread? Here is the direct link to the English file in the merge request so you do not need to scroll in this thread: [7655bc2](https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/commit/7655bc237c6e7816641cdda0f961bb5c460b8683) <[7655bc2](https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/commit/7655bc237c6e7816641cdda0f961bb5c460b8683)> — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <[#146 (comment)](https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/pull/146#issuecomment-487771732)>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AG427GE5D3V3IMLCHKHXFNDPS54CDANCNFSM4HITBX7A>.
> Jean-Christophe Helary ----------------------------------------------- http://mac4translators.blogspot.com <http://mac4translators.blogspot.com/> @brandelune

Any news? :)

brandelune commented:
Shane,

Yes, news :)

The English HTML is updated but the corresponding json file located here:

https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/blob/master/WebContent/resources/locales/en/translation.json

is not.

My understanding is that translations are based on the contents of that json file, so just to make sure all the stakeholders are on the same line it would be better to update it first and then translated it.

Do you want me to updage the English json file and proceed with the Japanese translation ?

Jean-Christophe 

> On May 7, 2019, at 14:40, Shane Coughlan <notifications@github.com> wrote:
> 
> Definitely ! I'll check that later today ! JC
> On Apr 30, 2019, at 7:54, Shane Coughlan @.*** @.***>> wrote: @goneall <https://github.com/goneall> https://github.com/goneall <https://github.com/goneall>, thanks! @brandelune <https://github.com/brandelune> https://github.com/brandelune <https://github.com/brandelune>, do we still have translation available? If so, can you update https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/tree/master/WebContent/resources/locales/ja <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/tree/master/WebContent/resources/locales/ja> https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/tree/master/WebContent/resources/locales/ja <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/tree/master/WebContent/resources/locales/ja> as a pull request to reflect the new text in this thread? Here is the direct link to the English file in the merge request so you do not need to scroll in this thread: 7655bc2 <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/commit/7655bc237c6e7816641cdda0f961bb5c460b8683> <7655bc2 <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/commit/7655bc237c6e7816641cdda0f961bb5c460b8683>> — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <#146 (comment) <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/pull/146#issuecomment-487771732>>, or mute the thread https://github.com/notifications/unsubscribe-auth/AG427GE5D3V3IMLCHKHXFNDPS54CDANCNFSM4HITBX7A <https://github.com/notifications/unsubscribe-auth/AG427GE5D3V3IMLCHKHXFNDPS54CDANCNFSM4HITBX7A>.
> Jean-Christophe Helary ----------------------------------------------- http://mac4translators.blogspot.com <http://mac4translators.blogspot.com/> http://mac4translators.blogspot.com/ <http://mac4translators.blogspot.com/> @brandelune <https://github.com/brandelune>
> Any news? :)
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/pull/146#issuecomment-489916132>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AG427GF7ZYB52BADSRGZOZTPUEI6DANCNFSM4HITBX7A>.
> 

Jean-Christophe Helary
-----------------------------------------------
http://mac4translators.blogspot.com @brandelune

shanecoughlan commented:
Hi JC!

Bringing Gary into CC.

I have now updated both files (and corrected some errors) in the “suggested-front-page” branch!

Shane

> On May 7, 2019, at 17:58, Jean-Christophe Helary <notifications@github.com> wrote:
> 
> Shane,
> 
> Yes, news :)
> 
> The English HTML is updated but the corresponding json file located here:
> 
> https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/blob/master/WebContent/resources/locales/en/translation.json
> 
> is not.
> 
> My understanding is that translations are based on the contents of that json file, so just to make sure all the stakeholders are on the same line it would be better to update it first and then translated it.
> 
> Do you want me to updage the English json file and proceed with the Japanese translation ?
> 
> Jean-Christophe 
> 
> > On May 7, 2019, at 14:40, Shane Coughlan <notifications@github.com> wrote:
> > 
> > Definitely ! I'll check that later today ! JC
> > On Apr 30, 2019, at 7:54, Shane Coughlan @.*** @.***>> wrote: @goneall <https://github.com/goneall> https://github.com/goneall <https://github.com/goneall>, thanks! @brandelune <https://github.com/brandelune> https://github.com/brandelune <https://github.com/brandelune>, do we still have translation available? If so, can you update https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/tree/master/WebContent/resources/locales/ja <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/tree/master/WebContent/resources/locales/ja> https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/tree/master/WebContent/resources/locales/ja <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/tree/master/WebContent/resources/locales/ja> as a pull request to reflect the new text in this thread? Here is the direct link to the English file in the merge request so you do not need to scroll in this thread: 7655bc2 <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/commit/7655bc237c6e7816641cdda0f961bb5c460b8683> <7655bc2 <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/commit/7655bc237c6e7816641cdda0f961bb5c460b8683>> — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <#146 (comment) <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/pull/146#issuecomment-487771732>>, or mute the thread https://github.com/notifications/unsubscribe-auth/AG427GE5D3V3IMLCHKHXFNDPS54CDANCNFSM4HITBX7A <https://github.com/notifications/unsubscribe-auth/AG427GE5D3V3IMLCHKHXFNDPS54CDANCNFSM4HITBX7A>.
> > Jean-Christophe Helary ----------------------------------------------- http://mac4translators.blogspot.com <http://mac4translators.blogspot.com/> http://mac4translators.blogspot.com/ <http://mac4translators.blogspot.com/> @brandelune <https://github.com/brandelune>
> > Any news? :)
> > 
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/pull/146#issuecomment-489916132>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AG427GF7ZYB52BADSRGZOZTPUEI6DANCNFSM4HITBX7A>.
> > 
> 
> Jean-Christophe Helary
> -----------------------------------------------
> http://mac4translators.blogspot.com @brandelune
> 
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

goneall commented:
@shanecoughlan This looks ready to go - shall I go ahead and merge?

goneall commented:
@brandelune @shanecoughlan Is there a separate PR for the Japanese translations?  I noticed that only the english translation was included in this PR.

brandelune commented:
That should be separate. Wait a bit please.

Jean-Christophe 

> On May 23, 2019, at 3:02, goneall <notifications@github.com> wrote:
> 
> @brandelune <https://github.com/brandelune> @shanecoughlan <https://github.com/shanecoughlan> Is there a separate PR for the Japanese translations? I noticed that only the english translation was included in this PR.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/pull/146?email_source=notifications&email_token=AG427GCM2EY6HAYOCH7WWXTPWWDCHA5CNFSM4HITBX7KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODV72TFY#issuecomment-494905751>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AG427GBSRYL2PSCXQLHA67DPWWDCHANCNFSM4HITBX7A>.
> 

Jean-Christophe Helary
-----------------------------------------------
http://mac4translators.blogspot.com @brandelune

goneall commented:
Hi J C,

 

Thanks for the update.  Just let me know when it is ready and I’ll roll a new release with the updated text.


Gary

 

From: Jean-Christophe Helary <notifications@github.com> 
Sent: Wednesday, May 22, 2019 4:05 PM
To: OpenChain-Project/Online-Self-Certification-Web-App <Online-Self-Certification-Web-App@noreply.github.com>
Cc: goneall <gary@sourceauditor.com>; Mention <mention@noreply.github.com>
Subject: Re: [OpenChain-Project/Online-Self-Certification-Web-App] added updated index.html (#146)

 

That should be separate. Wait a bit please.

Jean-Christophe 

> On May 23, 2019, at 3:02, goneall <notifications@github.com <mailto:notifications@github.com> > wrote:
> 
> @brandelune <https://github.com/brandelune> @shanecoughlan <https://github.com/shanecoughlan> Is there a separate PR for the Japanese translations? I noticed that only the english translation was included in this PR.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/pull/146?email_source=notifications <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/pull/146?email_source=notifications&email_token=AG427GCM2EY6HAYOCH7WWXTPWWDCHA5CNFSM4HITBX7KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODV72TFY#issuecomment-494905751> &email_token=AG427GCM2EY6HAYOCH7WWXTPWWDCHA5CNFSM4HITBX7KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODV72TFY#issuecomment-494905751>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AG427GBSRYL2PSCXQLHA67DPWWDCHANCNFSM4HITBX7A>.
> 

Jean-Christophe Helary
-----------------------------------------------
http://mac4translators.blogspot.com @brandelune





—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/pull/146?email_source=notifications&email_token=AAC4KPC6GX2WLHQEVZRWJ4LPWXGRTA5CNFSM4HITBX7KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODWATGBA#issuecomment-495006468> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AAC4KPF4LF5B3JWANDW3RLDPWXGRTANCNFSM4HITBX7A> .  <https://github.com/notifications/beacon/AAC4KPEE4QZ3ZWCANFHNMTLPWXGRTA5CNFSM4HITBX7KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODWATGBA.gif>

theopenchainproject commented:
Hi @brandelune 
Can you let me know the status on this? We really need to go live.

brandelune commented:
Working on it now.

shanecoughlan commented:
@brandelune is this completed? Really want to roll out.

brandelune commented:
It will be up *by* Monday. Really sorry for that.

shanecoughlan commented:
> It will be up _by_ Monday. Really sorry for that.

No problem! It will make a good Monday announcement.

goneall commented:
Note that once the code is checked in, it will take me some time to integrate, test, and deploy the code to the production server.  If nothing goes wrong, it would take about a day but I would set expectations of a week in case something goes wrong and to give me some flexibility.

brandelune commented:
I was on my way to committing the Japanese file for:
[https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/blob/master/WebContent/resources/locales/en/translation.json](https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/blob/master/WebContent/resources/locales/en/translation.json)

But I found that the Japanese had already been updated pretty much at the time we started this thread... Which is weird since Noriko worked on the translation last night...

Also, I'm finding that:

[https://github.com/OpenChain-Project/conformance-questionnaire/blob/master/questionnaire-en.json](https://github.com/OpenChain-Project/conformance-questionnaire/blob/master/questionnaire-en.json)

and

[https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/blob/master/resources/messages.properties](https://github.com/OpenChain-Project/Online-Self-Certification-Web-App/blob/master/resources/messages.properties)

Have had modifications that were not registered on my side.

I guess that's a sync issue on my side. Noriko is coming back later this afternoon so we'll be working on that this evening. Some messages are pretty cryptic so we'll need Gary's help.

brandelune commented:
@goneall Would you mind clarify a few strings ?

in messages.properties
1) Invalid specification version {0} language {1} due to {2}.
2) Duplication section name {0}.

Thank you in advance.

goneall commented:
@brandelune 

> Invalid specification version {0} language {1} due to {2}.

This is an error  when uploading a new version of the questionnaire.json file.  Version is the specification version and language is the languge code (e.g. "en", "ru").  The text in {2} is an error message which is already translated.

> Duplication section name {0}.

This message is generated as an error generated when uploading a new questionnaire when there is a section name used in two places.  {0} is the name of the section exactly as represented in the questionnaire.json file.  This is also a good example of what may be included in {2} above.

shanecoughlan commented:
Hi guys. Can we deploy?

goneall commented:
@shanecoughlan @brandelune Have the strings been updated?  If so, I can merge/deploy this week.

brandelune commented:
Ok, I committed all the updates, we should be good now !

goneall commented:
@brandelune Where are the changes committed?  I don't see them in this PR or in any other PR.

I'm going to go ahead and merge this and create a separate issue to track the localization.  If you could create a PR, I'll merge and deploy.
